CS4140/chess
# Add piece inventories

Test procedure:

$ iex -S mix
> user = Chess.Accounts.get_user_by_email(your_email)
> Chess.Repo.insert(%Chess.Piece{type: :rook, owner: user})
> Chess.Repo.insert(%Chess.Piece{type: :dragon, owner: user})
> Chess.Repo.insert(%Chess.Piece{type: :wizard, owner: user})

$ mix phx.server

Navigate to localhost:4040/prepare and arrange pieces

$ iex -S mix
> Chess.Piece.Inventory.get(user)

Confirm "origin" field is equal to the space the piece was moved to in the browser.
